lms/djangoapps/courseware/management/commands/export_course.py

```diff
@@ -37,7 +37,7 @@ def handle(self, *args, **options):
 
         results = self._get_results(filename) if pipe_results else None
 
-        return results
+        self.stdout.write(results, ending="")
```

Comment:
The recommendation in the django docs is to use self.stdout.write().  and the value returned from handle() is printed that way.